onnxscript/rewriter/__init__.py

```diff
@@ -31,6 +32,7 @@
     gemm_to_matmul_add.rule,  # type: ignore[has-type]
     *cast_constant_of_shape.rules.rules,
     *collapse_slices.rules.rules,
+    *erfgelu.rules.rules,
```

Comment:
```suggestion
```

this is ort specific right?

Reply:
Thats right, might be a better idea to move erfgelu fusion rules under ort-fusions folder then